docs/pages/eas-update/deployment-patterns.mdx

```diff
@@ -83,18 +83,29 @@ This flow is great for managing versioned releases. Here are the parts of the de
 
 ### Explanation of flow
 
-1. Develop a project locally and test changes in Expo Go.
-2. Create builds with channels named "production", which will eventually get reviewed and become available on app stores. Create another set of builds with channels named "staging", which will be used for testing on TestFlight and the Play Store Internal Track.
+1. Develop a project locally and test changes in Expo Go or a [Development Build](https://docs.expo.dev/develop/development-builds/introduction/).
+2. Create builds with channels named "production-rtv-1" (indicating a channel with a runtime version 1), which will eventually get reviewed and become available on app stores. Create another set of builds with channels named "staging", which will be used for testing on TestFlight and the Play Store Internal Track.
 3. Set up `expo-github-action` to publish updates when merging commits to branches.
 4. Merge changes into a branch named "version-1.0".
 5. Use the website or EAS CLI to point the "staging" channel at the EAS Update branch "version-1.0". Test the update by opening the apps on TestFlight and the Play Store Internal Track.
-6. When ready, use the website or EAS CLI to point the "production" channel at the EAS Update branch "version-1.0".
-7. Then, create another GitHub branch named "version-1.1", and merge commits until the new features and fixes are ready. When they are, use the website or EAS CLI to point the "staging" channel at the EAS Update branch "version-1.1".
-8. When ready, use the website or EAS CLI to point the "production" channel at the EAS Update branch "version-1.1".
+6. When ready, use the website or EAS CLI to point the "production-rtv-1" channel at the EAS Update branch "version-1.0".
+7. Then, there are two update scenarios you may encounter:
+    - A new release does not require a new runtime version:
+        1. Create another GitHub branch named "version-1.1".
```

Comment:
```suggestion
        0. Your runtime version of `1` remains unchanged. 
        1. Create another GitHub branch named "version-1.1".
```
Just to make it clear, since we don't conform to semver. Otherwise, I am afraid users may think it's possible to change their rtv from `1.0` to `1.1` and expect the same update to be delivered

Reply:
How about just using version strings like "1" and "2" instead of "1.1"?